src/Tribe/Service_Providers/Context.php

```diff
@@ -378,6 +379,22 @@ static function () {
 						Tribe__Context::QUERY_VAR        => [ 'tribe_view', 'eventDisplay' ],
 					],
 				],
+				'latest_event_date'    => [
+					'read'  => [
+						Tribe__Context::TRIBE_OPTION => [ 'latest_date' ]
+					],
+					'write' => [
+						Tribe__Context::TRIBE_OPTION => [ 'latest_date' ]
+					],
+				],
+				'earliest_event_date'  => [
+					'read'  => [
+						Tribe__Context::TRIBE_OPTION => [ 'earliest_date' ]
+					],
+					'write' => [
+						Tribe__Context::TRIBE_OPTION => [ 'earliest_date' ]
```

Comment:
* :no_entry_sign: Each line in an array declaration must end in a comma
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 8d129b78152924b9df18cde876492fa7</sub>